mm2src/coins/Cargo.toml

```diff
@@ -103,9 +103,10 @@ uuid = { version = "1.2.2", features = ["fast-rng", "serde", "v4"] }
 # We don't need the default web3 features at all since we added our own web3 transport using shared HYPER instance.
 web3 = { git = "https://github.com/KomodoPlatform/rust-web3", tag = "v0.19.0", default-features = false }
 zbase32 = "0.1.2"
-zcash_client_backend = { git = "https://github.com/KomodoPlatform/librustzcash.git", tag = "k-1.3.0" }
-zcash_primitives = { features = ["transparent-inputs"], git = "https://github.com/KomodoPlatform/librustzcash.git", tag = "k-1.3.0" }
-zcash_proofs = { git = "https://github.com/KomodoPlatform/librustzcash.git", tag = "k-1.3.0" }
+zcash_client_backend = { git = "https://github.com/KomodoPlatform/librustzcash.git", rev = "8f4b5e1" }
+zcash_extras = { git = "https://github.com/KomodoPlatform/librustzcash.git", rev = "8f4b5e1" }
+zcash_primitives = { features = ["transparent-inputs"], git = "https://github.com/KomodoPlatform/librustzcash.git", rev = "8f4b5e1" }
+zcash_proofs = { git = "https://github.com/KomodoPlatform/librustzcash.git", rev = "8f4b5e1" }
```

Comment:
Just a reminder that the librustzcash PR https://github.com/KomodoPlatform/librustzcash/pull/8 needs to be reviewed, merged and released before merging this PR.

Reply:
@borngraced can you please use the latest librustzcash release that includes async WalletDB https://github.com/KomodoPlatform/librustzcash/releases/tag/k-1.4.0
Please also resolve conflicts. I will do a last review of this PR after this.

Reply:
done @shamardy